utils/getters.js

```diff
@@ -23,7 +23,7 @@ export const getLocalizedCollection = (
   articleIndex,
   name: localize(name, lang),
   description: localize(description, lang),
-  articles: articles.map(a => getLocalizedArticle(a, lang)), // eslint-disable-line no-use-before-define
+  articles: articles?.map(a => getLocalizedArticle(a, lang)), // eslint-disable-line no-use-before-define
```

Comment:
maybe add optional param `omitArticles`?

Reply:
How is it better than just omitting them if they are not present?